ESX/Inventory HUD/README.md

```diff
@@ -59,6 +59,126 @@ TriggerServerEvent('dp_inventory:updateAmmoCount', GetHashKey(weaponModel), wepI
 local addToSlote = true
 TriggerEvent('dp_inventory:addCurrentWeapon', weaponModel, wepInfo, addToSlote)
 ```
+
+For police job you ca do like that and when they buy it its going to have a unique weapon id.
+This is clined line 1092
```

Comment:
This line is for dragging
https://github.com/esx-framework/esx_policejob/blob/9ceb66a591bca41ab10786bc795ebf04e52e2877/client/main.lua#L1092